packages/api/emails/hipaa/html.pug

```diff
@@ -70,11 +73,12 @@ div.main
     | will not be extended.
   p
     | Lastly, please understand that this application process
-    | does not guarantee that the nominated individual/family will receive a grant from Keep Swimming Foundation.
-    | It is imperative that the
+    span.text__bold.text__italic  does not guarantee that the nominated individual/family will receive a grant from Keep Swimming Foundation.
+    |  It is imperative that the
     | nominated individual/family does not rely on these funds and that the nominated individual/family
     | understands that they will only be contacted by Keep Swimming Foundation
-    | after the nearest quarterly
+    span.text__bold.text__italic  after
```

Comment:
Same here about the italics

Reply:
> It also looks like only the English version got updated but I think the Spanish version needs the styling as well.

per the template, the spanish version doesn't have new styling.

Reply:
https://docs.google.com/document/d/1SIA3SYM24c8y1ivenQrNn2mxD3XqB4Dp/edit